extensions/sql-migration/src/wizard/targetSelectionPage.ts

```diff
@@ -163,6 +164,10 @@ export class TargetSelectionPage extends MigrationWizardPage {
 					if (targetMi && targetMi.properties?.state.toLowerCase() !== 'Ready'.toLowerCase()) {
 						errors.push(constants.MI_NOT_READY_ERROR(targetMi.name, targetMi.properties?.state));
 					}
+					if (this.migrationStateModel.tdeMigrationConfig.shouldAdsMigrateCertificates() &&
+						this.migrationStateModel.tdeMigrationConfig.lastTdeMigrationResult().state !== TdeMigrationState.Succeeded) {
+						errors.push(constants.TDE_MIGRATE_REQUIRED);
```

Comment:
I am putting this here as a safeguard in case "Next" button doesn't become disabled as intended.  In a normal case, this error should never be surfaced.